### PR TITLE
Define more for libkernel and net.

### PR DESCRIPTION
**Pull request by OSM-Made**

### CODE REVIEW

**Comment by kiwidoggie** [comment]
One of the samples has broken with this change, can you look into it? https://github.com/OpenOrbis/OpenOrbis-PS4-Toolchain/actions/runs/3732893067/jobs/6332947844#step:33:431

**Comment by OSM-Made** [comment]
Seems it failed for an unrelated issue this time. It failed to fetch musl. Not sure if there is a way to kick another build with GitHub actions or not.

**Comment by illusion0001** [comment]
Rebase can trigger a new run

File: include/orbis/_types/net.h [hunk 1]
```diff
@@ -1,11 +1,108 @@
...
+
+#define ORBIS_NET_INET_ADDRSTRLEN     16
+
+/* multicast */
+#define ORBIS_NET_IP_DEFAULT_MULTICAST_TTL   1
+#define ORBIS_NET_IP_DEFAULT_MULTICAST_LOOP  1
+
+/* flags */
+#define ORBIS_NET_MSG_PEEK            0x00000002
+#define ORBIS_NET_MSG_WAITALL         0x00000040
```
Comment (bucanero) [inline include/orbis/_types/net.h:101]: if @OSM-Made values are right, then we need to update `MSG_xxx` definitions under socket.h. For example,
```
/include/sys/socket.h:306:#define MSG_WAITALL   0x0100
```

some other discrepancies (I didn't check everything, but I'm sure other values will need fix too), like:
```c
#define SO_RCVTIMEO     0x1006
#define SO_SNDTIMEO     0x1005
```

**Pull request closed without merging**